tests/cmdline/commands/test_presto.py

```diff
@@ -28,23 +28,32 @@ def get_profile_names(self):
 
 
 @pytest.mark.usefixtures('empty_config')
-@pytest.mark.parametrize('with_broker', (True, False))
-def test_presto(run_cli_command, with_broker, monkeypatch):
-    """Test the ``verdi presto``."""
+def test_presto_without_rmq(pytestconfig, run_cli_command, monkeypatch):
```

Comment:
I think it ended up being cleaner to split this test into two instead of parametrizing it.

Reply:
Meh, I see a lot of code duplication, but fine, I will let it slide :D

Reply:
Here's the alternative

```python
 @pytest.mark.usefixtures('empty_config')
   31 @pytest.mark.parametrize('with_broker', (pytest.param(True, marks=pytest.mark.requires_rmq), False))
   32 def test_presto(pytestconfig, run_cli_command, with_broker, monkeypatch):
   33     """Test the ``verdi presto``."""
   34     from aiida.brokers.rabbitmq import defaults
   35 
   36     if not with_broker:
   37         # Patch the RabbitMQ detection function to pretend it could not find the service
   38         monkeypatch.setattr(defaults, 'detect_rabbitmq_config', lambda: None)
   39 
   40     result = run_cli_command(verdi_presto)
   41     assert 'Created new profile `presto`.' in result.output
   42 
   43     with profile_context('presto', allow_switch=True) as profile:
   44         assert profile.name == 'presto'
   45         localhost = Computer.collection.get(label='localhost')
   46         assert localhost.is_configured
   47         if with_broker:
   48             assert profile.process_control_backend == 'core.rabbitmq'
   49         else:
   50             assert profile.process_control_backend is None
```

All I see is if statements and complexity. :stuck_out_tongue: (but honestly both feel fine)